packages/react/roving-focus/src/RovingFocusGroup.tsx

```diff
@@ -140,9 +143,11 @@ const RovingFocusGroupImpl = React.forwardRef<
         [setCurrentTabStopId]
       )}
       onItemShiftTab={React.useCallback(() => setIsTabbingBackOut(true), [])}
+      onFocusableItemAdd={React.useCallback(() => setFocusableItemsCount((n) => n + 1), [])}
+      onFocusableItemRemove={React.useCallback(() => setFocusableItemsCount((n) => n - 1), [])}
```

Comment:
```suggestion
      onFocusableItemAdd={React.useCallback(() => setFocusableItemsCount((prevCount) => prevCount + 1), [])}
      onFocusableItemRemove={React.useCallback(() => setFocusableItemsCount((prevCount) => prevCount - 1), [])}
```

Nit: We typically prefer to explicitly label previous values in setters

Reply:
Done.